benchmark/jsonschema/compiler_basic.cc

```diff
@@ -0,0 +1,25 @@
+#include <benchmark/benchmark.h>
+
+#include <sourcemeta/jsontoolkit/json.h>
+#include <sourcemeta/jsontoolkit/jsonschema.h>
+
+static void Compiler_Basic(benchmark::State &state) {
+  for (auto _ : state) {
+    const sourcemeta::jsontoolkit::JSON schema{
+        sourcemeta::jsontoolkit::parse(R"JSON({
+      "$schema": "https://json-schema.org/draft/2019-09/schema",
+      "properties": {
+        "foo": { "type": "string" },
+        "bar": { "type": "integer" }
+      }
+    })JSON")};
+
+    const auto compiled_schema{sourcemeta::jsontoolkit::compile(
```

Comment:
Google Benchmark has some macros that can prevent the compiler optimising these away. Can you take look? Otherwise, because you are not using this variable, some compiler configurations will get rid of the entire thing and we will not be benchmarking anything

Reply:
yeah sorry for all of this, TBH I just wrote it to have something that "runs" 😅